dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -48,6 +48,16 @@ func Single(args []string) error {
 	additionalFlags(singleCmd)
 	singleCmd.Parse(args[2:]) // error handling by flag.ExitOnError
 
+	if customUfsModuleFlag != "" {
+		customUfsModuleFlagArray := strings.Split(customUfsModuleFlag, ",")
+		if len(customUfsModuleFlagArray) == 3 {
+			ufsModules[customUfsModuleFlagArray[0]] =
```

Comment:
note that the key for `ufsModules` can be approximately derived from the `name` of the module. what do you think about simplifying the format of `customUfsModuleFlag` to `<name>,<maven args>`? for example, it would look like `-custom-ufs-module "hadoop-x.y,-pl underfs/hdfs -Pufs-hadoop-X -Dufs.hadoop.version=X.X.X"`

after checking that the string splits into 2 parts, set `ufsModules["ufs-"+customUfsModuleFlagArray[0]] = module{customUfsModuleFlagArray[0], true, customUfsModuleFlagArray[1]}`

Reply:
Good suggestion, done